lib/libzfs/libzfs_pool.c

```diff
@@ -3073,6 +3073,39 @@ zpool_vdev_offline(zpool_handle_t *zhp, const char *path, boolean_t istmp)
 	}
 }
 
+/*
+ * Remove the specified vdev. Called from zed on udev remove event.
+ */
+int
+zpool_vdev_remove_wanted(zpool_handle_t *zhp, const char *path)
```

Comment:
Both `zpool_vdev_remove_wanted()` and `zpool_vdev_remove()` are exported by libzfs and the same signature:
```C
/*
 * Remove the given device.
 */
int
zpool_vdev_remove(zpool_handle_t *zhp, const char *path)
```
Can you add some additional comments to differentiate what `zpool_vdev_remove_wanted()` does vs `zpool_vdev_remove()`?

Reply:
That's a very good point. ```zpool_vdev_remove_wanted()``` asynchronously removes the vdev whereas ```zpool_vdev_remove()``` does it synchronously. I will test it again to recall what issues I faced when using ```zpool_vdev_remove()``` and will add the comment accordingly.

Reply:
:+1:

Reply:
Just update the comment. Thanks!